src/c/PebbleAccelerometer.c

```diff
@@ -1,11 +1,63 @@
 #include <pebble.h>
 
 #define ACCEL_SAMPLE_RATE    ACCEL_SAMPLING_10HZ
+#define SAMPLES_PER_CALLBACK  1
+
+#define SYNC_BUFFER_SIZE      48
 
 static Window *s_main_window;
-static TextLayer *s_time_layer;
 static bool isCapturing = false;
 
+
+static TextLayer *s_time_layer;
+static int      syncChangeCount = 0;
+
+enum Pebble_Keys {
+  PP_KEY_CMD  = 128,
+  PP_KEY_X    = 1,
+  PP_KEY_Y    = 2,
+  PP_KEY_Z    = 3,
+};
+
+enum PebblePointer_Cmd_Values {
+  PP_CMD_INVALID = 0,
+  PP_CMD_VECTOR  = 1,
+};
+
+typedef struct {
+  uint64_t    sync_set;
+  uint64_t    sync_vib;
+  uint64_t    sync_missed;
+} sync_stats_t;
+
+static sync_stats_t   syncStats = {0, 0, 0};
+
+static bool isBlocked = false;;
+
+void out_sent_handler(DictionaryIterator *sent, void *context) {
+  // outgoing message was delivered
+  isBlocked = false;
+  APP_LOG(APP_LOG_LEVEL_INFO, "Booyah Baby!: outbox");
+}
+
+
+void out_failed_handler(DictionaryIterator *failed, AppMessageResult reason, void *context) {
+  // outgoing message failed
+  APP_LOG(APP_LOG_LEVEL_ERROR, "Message Sent Failed. Reason: %d", (int)reason);
+}
+
+
+void in_received_handler(DictionaryIterator *received, void *context) {
+  // incoming message received
+  APP_LOG(APP_LOG_LEVEL_INFO, "Booyah Baby! : inbox");
+}
+
+
+void in_dropped_handler(AppMessageResult reason, void *context) {
+  // incoming message dropped
+  APP_LOG(APP_LOG_LEVEL_ERROR, "Message dropped. Reason: %d", (int)reason);
+}
+
 static void main_window_load(Window *window)
 {
```

Comment:
so where are we putting braces? it seems to switch between the next line and the same line as the method name.  I don't care which is it as long as it is consistent